docs/tutorials/02_ephemeral_user.md

```diff
@@ -0,0 +1,89 @@
+Please see the Hello World Tutorial for initial setup of the Forsta Messenger Client. Complete code for this tutorial is available at:
+{@link https://github.com/ForstaLabs/forsta-messenger-client/tree/master/examples/ephemeral}
+
+The Forsta Messenger Client can be used with two different types of users:
+
+- Ephemeral - Short lived users created via API or by authenticating with the Forsta Ephemeral Chat token
+- Regular Users - Users generated via the API without an expiration or at {@link https://app.forsta.io}
+
+In this tutorial we will configure the client to create an ephemeral user who can chat with a regular
+user who has the tag @support.
+
+We will start with the following base code setting up the Forsta Messenger Client in a hidden div.
+This prevents the user from seeing the client until setup is complete.
+
+```html
+<html>
+  <head>
+    <script src="https://forstalabs.github.io/forsta-messenger-client/dist/forsta-messenger-client.min.js"></script>
+  </head>
+
+  <body>
+    <div id="my-messenger" style="width: 80%; height: 80%; visibility: hidden;"></div>
+  </body>
+
+  <script>
+    const myClient = new forsta.messenger.Client(document.getElementById('my-messenger'),
+      { orgEphemeralToken: 'TESTING' },
+      { });
+  </script>
+</html>
+
+```
+
+The Forsta Messenger Client includes several callbacks that can be configured when the
+client is initialized. In this case we will set the onLoaded callback to do the following:
+
+- Create a new conversation thread with a tagged users
+- Open the thread in the client
+- Display the client once the thread is available
+
+First, create the onload callback. It uses the threadMake command to generate a new thread
+between the ephemeral user and one or more users defined by their tags. For more information
+on using tags see {@link https://docs.forsta.io/docs/tag-expressions}. Once a thread has been
```

Comment:
Bonus points if that link could be made an `External` ref like the `ExchangePayload` and then we could convert any refs to `TagExpression` (and this one) to point to the external ref which would go to that link.  This way all internal refs to tag expression could be funneled through the same flow of "Common name" -> "Locally defined description of tag expression with link to detailed docs" -> "click away to external docs on full tag expression info".

Reply:
Done but will be in the next round of docs.